protostar-rust/src/main.rs

```diff
@@ -0,0 +1,107 @@
+use anyhow::{anyhow, Context, Result};
+use cairo_lang_protostar::test_collector::LinkedLibrary;
+use clap::Parser;
+use rust_test_runner_bindings::run_tests;
+use scarb_metadata::{Metadata, MetadataCommand, PackageId, PackageMetadata};
+use serde::Deserialize;
+use std::env::set_var;
+use std::path::PathBuf;
+
+const CONTRACT_TARGET: &str = "starknet-contract";
+
+#[derive(Parser, Debug)]
+struct Args {
+    // #[clap(required = true)]
+    // paths: Vec<PathBuf>,
+    #[arg(short, long)]
+    linked_libraries: Option<Vec<PathBuf>>,
```

Comment:
I assume this will be read from scarb.toml eventually? Will it be changed in this task #1938 ?

Reply:
It does that already. Take a look at `dependencies_for_package` in main.rs - the dependencies are resolved there and passed as linked libraries. I don't want to include this kind of logic "deeper" in the runner.